File: Questions/src/test/java/com/algorithm/dynamic_programming/BackpackTest.java
package com.algorithm.dynamic_programming;

import org.junit.Test;

import static junit.framework.TestCase.assertEquals;

public class BackpackTest {
    @Test
    public void test() {
        Backpack backpack = new Backpack();
        backpack.printDescription();
        assertEquals(10, backpack.backPack(11, new int[]{2, 3, 5, 7}));
    }
}

File: Questions/src/test/java/com/algorithm/dynamic_programming/BackpackTest.kt
package com.algorithm.dynamic_programming

import org.junit.Test

import junit.framework.TestCase.assertEquals

class BackpackTest {
    @Test
    fun test() {
        val backpack = Backpack()
        backpack.printDescription()
        assertEquals(10, backpack.backPack(11, intArrayOf(2, 3, 5, 7)))
    }
}
